tech.json

```diff
@@ -994,6 +994,23 @@
             "The ability to follow a shot to prevent it from despawning before hitting its target.",
             "This may include aiming between enemies or waiting for the shot to travel some distance before following it."
           ]
+        },
+        {
+          "name": "canAutoCancelWeapon",
+          "requires": [
+            {"or": [
+              "Missile",
+              "Super",
+              "PowerBomb",
+              "Grapple",
+              "XRayScope"
+            ]}
+          ],
```

Comment:
```suggestion
          "requires": [],
```
This feels a little weird to me, and I think it's fine being left off. Partially because it's not necessary, as strats that use it should have some requirements, such as an ammo cost. And partially because it's overly complex and yet still not completely accurate (i.e. you can't do it with 0/5 missiles and nothing else.)